### PR TITLE
Add BSP version to the names of output tars and wics

### PR DESCRIPTION
**Pull request by jsuhaas22**
There is a requirement to include the BSP versions in the names of the images we distribute. So include the version in the name.

### CODE REVIEW

**Comment by cshilwant** [comment]
@jsuhaas22 Ideally we should integrate https://github.com/TexasInstruments/ti-bdebstrap/pull/61 before this PR & then rebase on top to ensure the workflows tests `10.01.06` baseline tag.

Currently the workflows would still pick `10.00.07`

File: create-wic.sh [hunk 1]
```diff
@@ -83,17 +83,22 @@ source ${topdir}/scripts/common.sh
 
 validate_section "Build" ${BUILD} "${topdir}/builds.toml"
 
-if [ ! -f ${topdir}/build/${BUILD}/tisdk-debian-${BUILD}-boot.tar.xz ]; then
+
+distro_codename=($(read_build_config ${BUILD} distro_codename))
+bsp_version=($(read_bsp_config ${distro_codename} bsp_version))
+BUILD_FULL="${BUILD}-${bsp_version}"
```
Comment (glneo) [inline create-wic.sh:89]: Might be more clear what file is being used if you drop `BUILD_FULL` and just use `${BUILD}-${bsp_version}` directly below.

Comment (jsuhaas22) [reply create-wic.sh]: Updated the PR.

File: build.sh [hunk 2]
```diff
@@ -68,9 +70,9 @@ do
 
     validate_build ${machine} ${bsp_version} ${distro_codename}/${distro}.yaml
 
-    generate_rootfs ${distro} ${distro_codename} ${machine}
+    generate_rootfs ${distro} ${distro_codename} ${machine} ${distro_bsp_version}
```
[the remark this answers is not in the document]
Comment (jsuhaas22) [reply build.sh]: Made these changes @glneo

**Review by glneo — APPROVED** [review]

**Pull request merged**